packages/lex/lex-client/scripts/lex-build.mjs

```diff
@@ -19,6 +19,7 @@ Promise.all([
       'com.atproto.repo.putRecord',
       'com.atproto.repo.listRecords',
       'com.atproto.repo.uploadBlob',
+      'com.germnetwork.declaration',
```

Comment:
I'm not sure if this is necessary, now that `app.bsky.actor.profile` isn't directly referencing `com.germnetwork.declaration`